core/src/main/java/hudson/model/ParameterValue.java

```diff
@@ -305,6 +306,7 @@ public boolean isSensitive() {
      * If there's nothing that really fits the bill, the callee can return {@code this}.
      * @since 1.568
```

Comment:
it would be useful to have explanation of such occasions in Javadoc. Now it says `If there's nothing that really fits the bill, the callee can return {@code this}`